do-dont/doc/do-dont.md

```diff
@@ -57,14 +57,14 @@ export const api = () =>
 #### Do with component
 
 ```js preview-story
-export const do_with_component = () => html`<dockit-do>
+export const do_with_component = () => `<dockit-do>
   ${sampleButton}
   <ul slot="instructions">
     <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
     <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
     <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
-  </ul>
-</dockit-do>`;
+  </ul> </dockit-do
+>`;
```

Comment:
nitpick: smth went wrong here with the prettifier, good to fix it to show nicely formatted code to the user
sometimes prettier screws up HTML, so you need to do some manual formatting to assist it

Reply:
you have this below which is looking just great

```
export const dont_with_component = () => html`<dockit-dont>
  ${sampleButton}
  <ul slot="instructions">
    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
    <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
    <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
  </ul>
</dockit-dont>`;
```

yet smth is not great here, I'm pretty sure this can be done in the same way